ui/mirage/factories/service.js

```diff
@@ -19,16 +24,38 @@ export default Factory.extend({
       return null;
     }
   },
-  Connect: {
-    SidecarService: {
-      Proxy: {
-        Upstreams: [
-          {
-            DestinationName: dasherize(faker.hacker.noun()),
-            LocalBindPort: faker.random.number({ min: 5000, max: 60000 }),
-          },
-        ],
-      },
-    },
+
+  afterCreate(service, server) {
+    if (!service.namespace) {
+      const namespace = pickOne(server.db.jobs)?.namespace || 'default';
```

Comment:
This is randomly going to pick a job and assign the namespace of that job to this Service instead of taking the related job's namespace. Which could cause weird test related bugs down the line.